content/800-data-platform/100-accelerate/200-getting-started.mdx

```diff
@@ -82,26 +90,36 @@ Run the following command to install the Accelerate extension for Prisma Client:
 npm install @prisma/extension-accelerate
 ```
 
-If your Prisma version is below `4.16.1`, enable the Prisma Client extension feature:
+### 2.3. Generate Prisma Client for Accelerate
 
-```prisma
-/// prisma/schema.prisma
-generator client {
-  provider        = "prisma-client-js"
-  previewFeatures = ["clientExtensions"]
-}
-```
+If you're using Prisma version `5.2.0` or greater, Prisma Client will automatically determine how it should connect to the database depending on the protocol in the database connection string. If the connection string in the `DATABASE_URL` starts with `prisma://`, Prisma Client will try to connect to your database using Prisma Accelerate.
 
-### 2.3. Generate Prisma Client for Accelerate
+When using Prisma Accelerate in long-running application servers, such as a server deployed on AWS EC2, you can generate the Prisma Client by executing the following command:
 
-Run the following command to generate Prisma Client for Accelerate:
+```terminal
+npx prisma generate
+```
+
+When using Prisma Accelerate in a Serverless or an Edge application, we recommend you to run the following command to generate Prisma Client:
 
 ```terminal
-npx prisma generate --accelerate
+npx prisma generate --no-engine
 ```
 
+<Admonition type="info">
+
+The `--no-engine` flag prevents a Query Engine file from being included in the generated Prisma Client, this ensures the bundle size of your application remains small, which helps improve performance in your Serverless and Edge applications.
```

Comment:
We do not have any measurements that show it influences the performance in any significant way.
It does make the bundle smaller though, which means you have more space for other stuff before you get an error that your archive is too big.

Reply:
> which helps improve performance in your Serverless and Edge applications.
removing the line above!